docs/install/install-cave-beta.md

```diff
@@ -0,0 +1,165 @@
+
+---
+
+## Windows <i class="fa fa-windows"></i> 
+
+**Latest Version: 20.3.2-0.1**
+
+Currently the only installation available for Windows, is the [**Linux Virtual Machine**](#linux-virtual-machine).
+
+### Linux Virtual Machine
+
+Some CAVE functionality may be missing (ex: rendering RGB satellite images).
+
+#### System Requirements
+
+- [VMWare Workstation Player](https://www.vmware.com/products/workstation-player/workstation-player-evaluation.html) must be installed (free software):
+- For high definition monitors (4k), you will want to enable the high DPI setting for VMWare Workstation Player
+     1. Create a desktop shortcut for VMWare Workstation Player
+     1. Right-click the shortcut and select Properties
+     1. Open the Compatability Tab
+     1. Select the "Change high DPI settings" button
+     1. Check the "High DPI scaling ovveride" checkbox and choose "Application" in the enabled dropdown
+     
+      ![VMWare Workstation Player DPI Setting](../images/vmwareplayer-update-dpi.png)
+
+
+#### Download and Installation Instructions
+
+1. Download the zipped file containing the virtual machine: [**CentOS7-Unidata-CAVE-20.3.2-0.1** <i class="fa fa-download"></i>](https://downloads.unidata.ucar.edu/awips2/20.3.2/windows/CentOS7-Unidata-CAVE-20.3.2-0.1.zip)
+2. Unzip the folder.
+3. Open VMWare Player and go to **Player** > **File...** > **Open** and locate the folder that was created from the downloaded zipped file.  Select the file called **"CentOS 7 - Unidata CAVE 20.3.2-0.1.vmx"**.
+4. Run this new VM option.  If it asks if it's been moved or copied, select **"I Copied It"**.
+     - There will be a user in the Linux machine named "awips" and the password is "awips"
+     - The root password is "unidataAWIPS" if ever needed
+
+#### Run CAVE 
+
+Once inside the VM, to run CAVE either:
+
+- Use the desktop icon 
+- Use the terminal and type the command `cave`
+- Find the application in the Linux Desktop menu: Applications > Internet > AWIPS CAVE
+
+---
+
+## EDEX Connection
+
+Unidata and Jetstream2 have partnered to offer a EDEX data server in the cloud, open to the public.  Select the server in the Connectivity Preferences dialog, or enter **edex-beta.unidata.ucar.edu**.
+
+![EDEX in the cloud](../images/edex-beta-connection.png)
+
+---
+
+## Local Cache
+
+After connecting to an EDEX server, you will have a local directory named **caveData** which contains files synced from EDEX as well as a client-side cache for data and map resources.
+
+You can reset CAVE by removing the **caveData** directory and reconnecting to an EDEX server. Your local files have been removed, but if you are re-connecting to an EDEX server you have used before, the remote files will sync again to your local **~/caveData** (bundles, colormaps, etc.).
+
+- Linux: `/home/<user>/caveData/`
+- macOS: `/Users/<user>/Library/caveData/`
+- Windows: `C:\Users\<user>\caveData\`
+
+---
+
+## Functionality/Reporting
```

Comment:
As noted before, move this up higher.  And I would add a little more additional language.  Maybe this in total:

> This is a beta release, so we are aware that not all functionality is working as expected.  We ask you to please be aware of this and have similar expectations.
One noteworthy deficiency we are aware of is the radar menu has not been updated yet to reflect what is in version 18.  
If you come across issues/bugs/missing functionality, we also encourage you to [report it using this short form](https://forms.gle/aYUZSi78KU3ZYJ3q7).

Also, you can use the `href` tag, if the markdown syntax directly diverts the current page (like you already written).